src/main/java/com/palantir/docker/compose/DockerService.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ */
+
+package com.palantir.docker.compose;
+
+import com.palantir.docker.compose.connection.waiting.SingleServiceHealthCheck;
+import org.immutables.value.Value;
+
+@Value.Immutable
+public interface DockerService {
```

Comment:
I'm not attached to the name, it's just what we already had.  The aim was to encapsulate the information require to 'wait for a service' in one object.